test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenTransactSpecs.java

```diff
@@ -99,7 +102,11 @@ public static void main(String... args) {
 						uniqueTokenTxnWithSenderNotSigned(),
 						uniqueTokenTxnWithReceiverNotSigned(),
 						uniqueTokenTxnsAreAtomic(),
-						uniqueTokenDeletedTxn()
+						uniqueTokenDeletedTxn(),
+						balancesChangeOnTokenTransferWithFixedHbarCustomFees(),
+						transferFailsWithInsufficientBalanceForFixedCustomFees(),
```

Comment:
Very strange, these 2 tests passed in this branch but keep failing after merging to `master`.